test/test_formatters.rb

```diff
@@ -290,6 +289,10 @@ def test_format_file
     assert_equal content('formatter_gettext.po'), formatter.format_file('en')
   end
 
+  def test_determines_complex_language
```

Comment:
How about `test_determines_language_and_region` instead of `complex`?

We also have a few `test_deducts_language_[and_region_]from_resource_folder` tests. Do you think we should use similar names throughout the tests?